gnovm/pkg/integration/gno.go

```diff
@@ -68,6 +68,8 @@ func SetupGno(p *testscript.Params, buildDir string) error {
 			return fmt.Errorf("unable to create temporary home directory: %w", err)
 		}
 		env.Setenv("HOME", home)
+		// Avoids go command printing errors relating to lack of go.mod.
+		env.Setenv("GO111MODULE", "off")
```

Comment:
It's an idea. I guess more in general the thing is that we shouldn't use `go build`; because even if GO111MODULE=off, then it will try to use go's old depedency system.

This could eventually be fixed by https://github.com/gnolang/gno/pull/1730, which makes it so we don't need `go build` to get typecheck errors. I haven't managed to keep up with that one much, dealing with other priorities, alas. But I want to try to maybe wrap it up this week, before I go on holiday.

Reply:
Indeed, it would be even better without using `go build`.